doggos/knowledge/consequents/consequent.py

```diff
@@ -1,9 +1,11 @@
 from abc import ABC, abstractmethod
-from typing import Tuple
+from typing import Tuple, List
+
+from doggos.fuzzy_sets.degree.membership_degree import MembershipDegree
 
 
 class Consequent(ABC):
 
     @abstractmethod
-    def calculate_cut(self) -> Tuple[float, ...] or float:
+    def output(self, rule_firing: MembershipDegree) -> Tuple[List[float]] or List[float]:
```

Comment:
Type returned by output function should be List[MembershipDegree] or float.

Reply:
correct, for TakagiConsequent output is always float (both for Type-1 and IType-2)